libeos-payg/efi.c

```diff
@@ -196,7 +198,7 @@ efivarfs_write (const char  *name,
 
   /* It may not exist, and this will fail harmlessly */
   if (allow_overwrite)
-    clear_immutable (name);
+    clear_immutable (name, NULL);
```

Comment:
I also wondered about changing `clear_immutable` to operate on a file descriptor rather than opening it, changing its attribute, and closing it; only for this caller to open it again. But the test coverage here is nonexistent – it's not easy to mock efivarfs! –  so I preferred to leave it. The attempt to write will fail with (presumably) a meaningful error message anyway.